pkg/apis/infinispan/v2alpha1/zz_generated.openapi.go

```diff
@@ -5,9 +5,56 @@
 package v2alpha1
 
 import (
+	spec "github.com/go-openapi/spec"
 	common "k8s.io/kube-openapi/pkg/common"
 )
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
-	return map[string]common.OpenAPIDefinition{}
+	return map[string]common.OpenAPIDefinition{
+		"github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v2alpha1.Backup": schema_pkg_apis_infinispan_v2alpha1_Backup(ref),
+	}
+}
+
+func schema_pkg_apis_infinispan_v2alpha1_Backup(ref common.ReferenceCallback) common.OpenAPIDefinition {
```

Comment:
@rigazilla it seems to be you are generated code with different SDK version

Reply:
no SDK is ok.  I've run an unneeded `generate openapi`. thanks

Reply:
Oh, yes. I'm prefer to type just 'make codegen` operator-sdk ... to long for me :smile:

Reply:
ouch! you're right!